apps/web/src/components/user-tools/edit-or-invite/edit-or-invite.tsx

```diff
@@ -49,7 +49,16 @@ export function EditOrInvite({
   const hasUnrevised =
     unrevisedRevisions !== undefined && unrevisedRevisions > 0
 
-  const href = hasUnrevised ? getHistoryUrl(data.id) : getEditHref()
+  const isCourse =
+    data.type === UuidType.Course || data.type === UuidType.CoursePage
+
+  const href = isCourse
```

Comment:
Nit/Suggestion:

The ternary is quite pretty, however I would be inclined to do something like:

```
function getHref({ isCourse, data, hasUnrevised }) {
  if (!isCourse) return hasUnrevised ? getHistoryUrl(data.id) : getEditHref();
    
    return data.unrevisedCourseRevisions && data.unrevisedCourseRevisions > 0
      ? getHistoryUrl(data.courseId ?? data.id)
      : getEditHref();
}

// Usage
const href = getHref({ isCourse, data, hasUnrevised });
```

Reply:
(using ` ``` tsx` gives you syntax highlighting :)

Reply:
thanks for the suggestion!
I'll leave it for now.